src/components/shortcuts.js

```diff
@@ -205,7 +210,8 @@ function executeAction(card, target, action) {
             playbackManager.play({
                 ids: [playableItemId],
                 startPositionTicks: startPositionTicks,
-                serverId: serverId
+                serverId: serverId,
+                queryOptions: userSettings.getSortValuesLegacy(sortParentId, 'SortName')
```

Comment:
here too

Reply:
Are you sure about this one? I tried removing it and the photos sorting stops working :-/

Thank you for the feedback :)